src/childwindow.js

```diff
@@ -50,6 +43,9 @@ module.exports = (function () {
             this.triggerEvent('load');
             $(event.target).off(event);
         }, this));
+        message.on('widget-detection', function () {
```

Comment:
Child window has't been created at this moment, so you can't set event listener for it